pkg/cloudprovider/gcp/block_store.go

```diff
@@ -142,10 +162,20 @@ func (b *blockStore) CreateVolumeFromSnapshot(snapshotID, volumeType, volumeAZ s
 	}
 
 	if isMultiZone(volumeAZ) {
+		// Zone the volume is in
 		volumeRegion, err := parseRegion(volumeAZ)
 		if err != nil {
 			return "", err
 		}
+
+		// URLS for zones that the volume is replicated to within GCP
```

Comment:
s/URLS/URLs since you are going to be changing things :)

Reply:
Changed